src/rtc/geogrid.py

```diff
@@ -298,49 +345,67 @@ def generate_geogrids_from_db(bursts, geo_dict, dem, burst_db_file):
         Dict containing bursts geogrids indexed by burst_id
     '''
 
-    # TODO use `dem_raster` to update `epsg`, if not provided in the runconfig
-    # dem_raster = isce3.io.Raster(dem)
-
     # Unpack values from geocoding dictionary
-    epsg_runconfig = geo_dict['output_epsg']
-    xmin_runconfig = geo_dict['top_left']['x']
-    ymax_runconfig = geo_dict['top_left']['y']
-    x_spacing = geo_dict['x_posting']
-    y_spacing_positive = geo_dict['y_posting']
-    xmax_runconfig = geo_dict['bottom_right']['x']
-    ymin_runconfig = geo_dict['bottom_right']['y']
-    x_snap = geo_dict['x_snap']
-    y_snap = geo_dict['y_snap']
+    mosaic_geogrid_dict = mosaic_dict['mosaic_geogrid']
+    epsg_mosaic = mosaic_geogrid_dict['output_epsg']
+    xmin_mosaic = mosaic_geogrid_dict['top_left']['x']
+    ymax_mosaic = mosaic_geogrid_dict['top_left']['y']
+    x_spacing_mosaic = mosaic_geogrid_dict['x_posting']
+    y_spacing_positive_mosaic = mosaic_geogrid_dict['y_posting']
+    xmax_mosaic = mosaic_geogrid_dict['bottom_right']['x']
+    ymin_mosaic = mosaic_geogrid_dict['bottom_right']['y']
+    x_snap_mosaic = mosaic_geogrid_dict['x_snap']
+    y_snap_mosaic = mosaic_geogrid_dict['y_snap']
+
+
```

Comment:
```suggestion
```
Nit pick. Leave the blank like as it is when you think it's absolutely necessary.